lib/db.js

```diff
@@ -900,6 +908,15 @@ Db.prototype.watch = function(pipeline, options) {
   return new ChangeStream(this, pipeline, options);
 };
 
+/**
+ * Return the db logger
+ * @method
+ * @return {Logger} return the db logger
+ */
+Db.prototype.getLogger = function() {
```

Comment:
I like this approach of making everything that has a logger expose this. I am wondering if it should be a public method / documented. @mbroadst @kvwalker thoughts?

Reply:
I don't think we should document it, it's generally only helpful for us it seems (providing a common means of returning a logger that might live in different places). From a user's perspective, they should pass in a logger at the top level and that's all they should ever have to worry about.

Reply:
I agree that we should not document it since I'm not sure that users would use it.

Reply:
So should we go ahead and @ignore the comments for getLogger?